tests/statsbombpy_test/test_sb.py

```diff
@@ -37,10 +37,6 @@ def test_matches(self):
         self.assertIsInstance(matches, pd.DataFrame)
 
         matches = sb.matches(competition_id=11, season_id=1)
-        self.assertEquals(
-            matches.query("match_id == 9695")["home_managers"].iloc[0],
-            "Juan Rubén Uría Corral, Marcelino García Toral",
-        )
```

Comment:
Why did you remove this test ?

Reply:
That test failed because they changed the underlying data to only have one manager. I have pushed another commit to replace the test with another match where there's two managers.